setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPL
+classifier =
+    Development Status :: 3 - Alpha
+    Topic :: Utilities
+    License :: OSI Approved :: GNU General Public License 2 (GPL2)
```

Comment:
```
"License :: OSI Approved :: GNU General Public License 2 (GPL2)" != "License :: OSI Approved :: GNU General Public License v2 (GPLv2)"
```
missing 'v' infront of '2', the PYPI will complain about it if you publish package.